pkg/antctl/transform/networkpolicy/transform.go

```diff
@@ -63,35 +99,14 @@ func Transform(reader io.Reader, single bool, opts map[string]string) (interface
 	)(reader, single)
 }
 
-const sortByEffectivePriority = "effectivePriority"
-
-// Compute a tierPriority value in between the application tier and the baseline tier,
-// which can be used to sort all policies by tier.
-var effectiveTierPriorityK8sNP = (networkpolicy.DefaultTierPriority + networkpolicy.BaselineTierPriority) / 2
-
-type NPSorter struct {
-	networkPolicies []cpv1beta.NetworkPolicy
-	sortBy          string
-}
-
 func (nps *NPSorter) Len() int { return len(nps.networkPolicies) }
 func (nps *NPSorter) Swap(i, j int) {
 	nps.networkPolicies[i], nps.networkPolicies[j] = nps.networkPolicies[j], nps.networkPolicies[i]
 }
 func (nps *NPSorter) Less(i, j int) bool {
 	switch nps.sortBy {
-	case sortByEffectivePriority:
-		var ti, tj int32
-		if nps.networkPolicies[i].TierPriority == nil {
-			ti = effectiveTierPriorityK8sNP
-		} else {
-			ti = *nps.networkPolicies[i].TierPriority
-		}
-		if nps.networkPolicies[j].TierPriority == nil {
-			tj = effectiveTierPriorityK8sNP
-		} else {
-			tj = *nps.networkPolicies[j].TierPriority
-		}
+	case "effectivePriority":
```

Comment:
Ditto, use a constant instead of a slice. Otherwise, you need to maintain two places when you need to update the flag.

Reply:
I think you can define the constant and use it here and also in the new slice. So you can avoid maintain two places if the name is changed.